### PR TITLE
don't die when no or a https image is found

### PR DESCRIPTION
**Pull request by bumi**
currently https image urls are not supported and node dies when we try to download through https. 
I've quickly added a check for https and it now uses the https module for those urls.
Also there was an error when no image is found at all. in this case I'm now just using a fixed default image (d'oh).

### CODE REVIEW

**Comment by ghost** [comment]
please pull! we need these fixes for our campfirez!

File: index.js [hunk 1]
```diff
@@ -7,19 +8,32 @@ server = require('node-router').getServer()
...
-    var result = JSON.parse(body)['responseData']['results'][0]['unescapedUrl']
-    cb(result)
-  })
+    result = JSON.parse(body)['responseData']['results'][0];
+    
+    if(result)
+      cb(result['unescapedUrl']);
+    else
+      cb("https://img.skitch.com/20110825-ewsegnrsen2ry6nakd7cw2ed1m.png");
+  });
```
Comment (tmm1) [inline index.js:19]: Can you remove all these semicolons?

File: index.js [hunk 2]
```diff
@@ -7,19 +8,32 @@ server = require('node-router').getServer()
...
 
 function download(match, output, addText){
   fetch(match, function(file){
-    var host = url.parse(file).hostname
-      , path = url.parse(file).pathname
-
-    request = http.get({host: host, path: path}, function(res){
+    if(!match) {
+      
+    }
```
Comment (tmm1) [inline index.js]: Empty conditional?

Comment (bumi) [inline index.js]: haha, ooppps.  ;)
guess that's a left over of another idea. is removed now.

File: index.js (hunk 1 above)
Comment (bumi) [inline index.js:19]: done.
we kept the semicolons in our projects. I don't know of that/any convention. is there one?

**Pull request merged**